src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -46,7 +46,11 @@ effective_io_concurrency = {{ postgresql_effective_io_concurrency }}
 # WRITE AHEAD LOG
 {% if postgresql_enable_archive and not is_pg_standby %}
 archive_mode = on
+{% if hot_standby_server %}
+archive_command = 'scp %p {{hot_standby_server}}:{{ postgresql_archive_path }}/%f'
```

Comment:
this is going to require no-password auth between the two servers. Are you going to set that up separately?

Reply:
It's already part of the setup process 
https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/deploy_postgres.yml#L60